script.js

```diff
@@ -0,0 +1,65 @@
+const hamburg = document.querySelector('.hamburg');
+const Menu = document.querySelector('.menu');
+
+hamburg.addEventListener('click', () => {
+  hamburg.classList.toggle('display');
+  Menu.classList.toggle('display');
+});
+
+document.querySelectorAll('menu-item').forEach((n) => n.addEventListener('click', () => {
+  hamburg.classList.toggle('display');
+  Menu.classList.toggle('display');
+}));
+
+const Teachers = [{
+  teacherImagee: 'img-folder/teacher1.jpg',
+  teachearName: 'Bella mwa',
+  teacherDomain: 'her,english teacher',
+  teacherAbout: 'bella is best teacher of english she is from USA.',
+},
+{
+  teacherImagee: 'img-folder/teacher2.jpg',
+  teachearName: 'Bertin kam',
+  teacherDomain: 'him,spanish teacher',
+  teacherAbout: 'bertin is best teacher of spanish he is from Spain.',
+},
+{
+  teacherImagee: 'img-folder/teacher3.jfif',
+  teachearName: 'Linda ker',
+  teacherDomain: 'her,Arbic teacher',
+  teacherAbout: 'Linda is best teacher of Arbic she is from Arabi.',
+},
+{
+  teacherImagee: 'img-folder/teacher4.jfif',
+  teachearName: 'Benjamin nyam',
+  teacherDomain: 'him,Lingala teacher',
+  teacherAbout: 'benjamin is best teacher of lingala he is from DRC.',
+},
+{
+  teacherImagee: 'img-folder/teacher5.jfif',
+  teachearName: 'kevin Zam',
+  teacherDomain: 'him,french teacher',
+  teacherAbout: 'kevin is best teacher of french he is from France.',
+},
+{
+  teacherImagee: 'img-folder/teacher6.jfif',
+  teachearName: 'Rachel Bim',
+  teacherDomain: 'her,Italiano teacher',
+  teacherAbout: 'Rachel is best teacher of Italiano she is from Italia.',
+},
+];
+
+const allCards = document.querySelector('.blocteacher');
+
+Teachers.forEach((cards) => {
+  const div = document.createElement('div');
+  div.innerHTML = `<div class="cardteach">
+           <img src="${cards.teacherImagee}" alt="">
+      <div class="picteach">
+            <h3>${cards.teachearName}</h3>
+            <p class="italic">${cards.teacherDomain}</p>
+            <p>${cards.teacherAbout}</p>
+      </div>
+    </div>`;
+  allCards.appendChild(div);
```

Comment:
- [ ] As mentioned by the previous reviewer, in the `Teachers` section in the mobile version the images are not aligned with its details please 🙏  make sure to align them in away that the content details of the teacher comes next to its image and doesn't owerflow

Your design:

<img width="378" alt="Screenshot 2023-03-16 at 9 43 37 PM" src="https://user-images.githubusercontent.com/44798044/225730772-cd2de660-7b7c-46d9-98da-caab698246e8.png">

Behance design:

<img width="349" alt="Screenshot 2023-03-17 at 12 21 42 AM" src="https://user-images.githubusercontent.com/44798044/225731075-06d7daba-a701-437b-b748-873e03fb7562.png">